spring-native/build.gradle

```diff
@@ -5,21 +5,13 @@ plugins {
     id 'org.graalvm.buildtools.native' version '0.9.25'
 }
 
-compileJava {
-    sourceCompatibility = "17"
-    targetCompatibility = "17"
-}
 description = 'OpenTelemetry Example for Spring native images'
 ext.moduleName = "io.opentelemetry.examples.native"
 
-dependencyManagement {
```

Comment:
The new code has broken the Spring native image example. To make it works, add

dependencyManagement {
    imports {
        mavenBom 'io.opentelemetry:opentelemetry-bom:1.29.0'
    }
}